Sources/RingPublishingTracking/Private/Tracking/Events/EventsService+Decorators.swift

```diff
@@ -34,11 +34,16 @@ extension EventsService {
         adAreaDecorator.updateApplicationAdvertisementArea(applicationAdvertisementArea: currentAdvertisementArea)
     }
 
-    func updateUserData(ssoSystemName: String, userId: String?, email: String?) {
-        let preparedEmail = email?.trimmingCharacters(in: .whitespacesAndNewlines).lowercased()
-        let sso = SSO(logged: Logged(id: userId, md5: preparedEmail?.md5()), name: ssoSystemName)
+    func updateUserData(userId: String?, email: String?) {
```

Comment:
W ramach pracy nad tym interfejsem dodałbym tę funkcje do wylogowania (bezparametrową), która pod spodem ustawia userId i userEmail na null

Reply:
jest, to co komentujesz nie jest publiczne